### PR TITLE
Solve edit command bug

### LINKED ISSUE #136: Edit students does not update tutorial json file
Will fix in next iteration

### PR DESCRIPTION
**Pull request by SamsonChew**
closes #136

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -38,8 +38,8 @@ public class EditCommand extends Command {
             + "[" + PREFIX_TUTORIALID + "TUTORIAL_ID]\n"
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_NAME + "Samson Chew "
-            + PREFIX_STUDENTID + "A1234567M"
-            + PREFIX_TUTORIALID + "1001";
+            + PREFIX_STUDENTID + "A1234567X "
+            + PREFIX_TUTORIALID + "T1001";
```
Comment (Reallyeasy1) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:42]: Good job

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -99,6 +99,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         // Update tutorial assignments if tutorial ID has changed
         if (!editedStudent.getTutorialId().equals(studentToEdit.getTutorialId())) {
+            model.unassignStudent(studentToEdit, studentToEdit.getTutorialId());
```
Comment (Reallyeasy1) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:102]: interesting use of SLAP

File: src/main/java/seedu/address/model/tut/TutorialList.java [hunk 3]
```diff
@@ -102,6 +102,24 @@ public void assignStudent(Student student, TutorialId tutorialId) {
...
+     * @param student Student to be unassigned
+     * @param tutorialId Target tutorial
+     */
+    public void unassignStudent(Student student, TutorialId tutorialId) {
+        requireNonNull(student);
+        requireNonNull(tutorialId);
+        for (Tutorial tutorial : tutorials) {
+            if (tutorial.getTutorialId().equals(tutorialId)) {
+                tutorial.deleteStudent(student);
+                break;
```
Comment (Reallyeasy1) [inline src/main/java/seedu/address/model/tut/TutorialList.java:117]: good use of break

File: src/test/java/seedu/address/logic/commands/AttendCommandTest.java [hunk 4]
```diff
@@ -252,6 +252,11 @@ public void assignStudent(Student student, TutorialId tutorialId) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public void unassignStudent(Student student, TutorialId tutorialId) {
+            throw new AssertionError("This method should not be called.");
```
Comment (tansanchian) [inline src/test/java/seedu/address/logic/commands/AttendCommandTest.java:257]: Good to see that assertion is used!

**Review by tansanchian — APPROVED** [review]
LGTM!

**Pull request merged**